lib/Plack/Middleware/RewriteShibEnv.pm

```diff
@@ -0,0 +1,32 @@
+use strict;
+use warnings;
+
+package Plack::Middleware::RewriteShibEnv;
+use parent 'Plack::Middleware';
+
+my $SHIB_ENV_VARS = {
+  'HTTP_X_SHIB_AUTHENTICATION_METHOD' => 'X-Shib-Authentication-Method',
```

Comment:
See comment above - I think we can just provide a list of environment variables to copy from the external environment; we don't need to change them to their HTTP header name form.

Reply:
@aelkiss do we have to be concerned with the Debian 9 business where CGI scripts got `Shib_Identity_Provider` but FastCGI got `Shib-Identity-Provider`?

Reply:
I don't think so, since we will be planning to run this via HTTP proxy rather than via FastCGI.